providers/cncf/kubernetes/src/airflow/providers/cncf/kubernetes/executors/kubernetes_executor.py

```diff
@@ -269,6 +275,26 @@ def execute_async(
         # try and remove it from the QUEUED state while we process it
         self.last_handled[key] = time.time()
 
+    def queue_workload(self, workload: workloads.ExecuteTask, session: Session | None) -> None:
+        ti = workload.ti
+        self.queued_tasks[ti.key] = workload
+
+    def _process_workloads(self, workloads: list[workloads.All]) -> None:
+        # Airflow V3 version
+        for w in workloads:
+            if not w.token:
+                w.token = "placeholder-token"
+            command = [w]
+            key = w.ti.key
+            queue = w.ti.queue
+
+            # TODO: setting to empty for now, needs to be revisited
+            executor_config = {}  # type: ignore[var-annotated]
```

Comment:
Can you expand further on what needs to be revisited here? Is there some blocker for supporting executor_config?